tasks.py

```diff
@@ -177,11 +177,5 @@ def before_install(group, python_version):
 
 @task
 def install(group):
-    # symlink, depending on whether it's docs or not
-    if group != 'docs':
-        cmd = 'python setup.py develop'
-    else:
-        cmd = 'python setup.py install'
-
-    # install
+    cmd = 'pip install -r dev-requirements.txt -e .'
```

Comment:
The if statement that's there needs to be there, because the coverage doesn't get properly computed if its a symlinked install. Can you have it do `pip install -r dev-requirements.txt .` if it's docs and otherwise install it with `-e`?